include/deal.II/lac/affine_constraints.h

```diff
@@ -705,6 +705,17 @@ class AffineConstraints : public Subscriptor
   bool
   can_store_line(const size_type line_n) const;
 
+  /**
+   * Return the index set describing which part of the degrees of freedom to
+   * which this object stores constraints are "locally owned". Typically,
+   * these would be the
+   * @ref GlossLocallyOwnedDoFs "locally owned degrees of freedom".
```

Comment:
```suggestion
   * @ref GlossLocallyOwnedDoF "locally owned degrees of freedom".
```
should make `doxygen` happy.